src/lp.jl

```diff
@@ -0,0 +1,174 @@
+#=
+Interface to lrs_solve_lp
+
+=#
+
+
+mutable struct LRSLinprogSolution
+    status::Symbol  # :Optimal, :Infeasible, :Unbounded, :UserLimit, or :Error
+    objval::Union{Nothing,Rational{BigInt}}
+    sol::Vector{Rational{BigInt}}
+    attrs::Dict
+end
+
+
+function setobj(P::Ptr{Clrs_dic}, Q::Ptr{Clrs_dat},
+                c::Vector{<:Real}, maximize::Bool)
+    row = Vector{Rational{BigInt}}(undef, unsafe_load(P).d+1)
+    sgn = maximize ? 1 : -1
+    field = maximize ? :maximize : :minimize
+    row[1] = 0
```

Comment:
`row[1] = zero(eltype(row))` might be better as `convert(Rational{BigInt}, 0)` will call a constructor that is not specific to zero and might be less appropriate